lbry/wallet/ledger.py

```diff
@@ -752,7 +752,11 @@ def broadcast(self, tx):
 
     async def resolve(self, accounts, urls, **kwargs):
         resolve = partial(self.network.retriable_call, self.network.resolve)
-        txos = (await self._inflate_outputs(resolve(urls), accounts, **kwargs))[0]
+        urls_copy = list(urls)
+        txos = []
+        while urls_copy:
+            batch, urls_copy = urls_copy[:500], urls_copy[500:]
```

Comment:
fyi, this will re-allocate memory for two new lists on each loop, if the lists are big this will probably use lots of RAM